packages/react/src/Timeline/Timeline.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import {StoryFn, Meta} from '@storybook/react'
+
+import {Timeline} from '.'
+
+export default {
+  title: 'Components/Timeline',
+  component: Timeline,
+} as Meta<typeof Timeline>
+
+const Template: StoryFn<typeof Timeline> = () => (
+  <Timeline>
+    <Timeline.Item>
+      <em>GitHub Codespaces</em> offers a complete dev environment in seconds, so you can code, build, test, and open
```

Comment:
@danielguillan - could we add another story into a Timeline.features.tsx, where we don't apply the `<em>` please? Given we have some branching logic there to change the colors, it would be great to get this under our visual regression test suite 🙏